sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -290,15 +309,24 @@ export class EventHubClient {
   }
 
   /**
-   * Creates an EventHubConsumer that can be used to receive events from the Event Hub for which this
-   * client was created.
+   * Creates an Event Hub consumer responsible for reading `EventData` from a specific Event Hub parition,
+   * and as a member of a specific consumer group.
```

Comment:
parition -> partition

> and **as a member** of a specific consumer group

That doesnt make much sense to me when reading it as part of the bigger sentence

Reply:
Updated!